my-use-case/src/components/__init__.py

```diff
@@ -0,0 +1,14 @@
+from kfp.v2.dsl import Dataset, Input, Output, Model, component
+from src.dependencies import PYTHON37, XGBOOST, SKLEARN, PANDAS
+
+BASE_IMAGE = "gcr.io/my-org/my-image"
```

Comment:
something to think about - what if we wanted multiple (different) container images for different components (with an overarching build script + CI/CD integrated nicely)

Reply:
yes, that's certainly a matter of scalability & flexibility. ideally components within shared-pipeline-components should have the possibility to have separate dep's or share dep's if required. eventually the same for my-use-case, however, typical DS projects could be satisfied with a single dep tree.